src/zenml/integrations/pycaret/__init__.py

```diff
@@ -0,0 +1,38 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Initialization of the PyCaret integration."""
+
+from zenml.integrations.constants import PYCARET
+from zenml.integrations.integration import Integration
+
+
+class PyCaretIntegration(Integration):
+    """Definition of PyCaret integration for ZenML."""
+
+    NAME = PYCARET
+    REQUIREMENTS = [
+        "pycaret",
```

Comment:
Wonder whether it's worth specifying >3.0.0 here? @fa9r thoughts?

Reply:
If it only works for `pycared>3.0.0` then yes otherwise no 😄

Reply:
@christianversloot I'm not familiar enough with `pycaret`'s API to have an opinion on this. From a brief perusal of their release notes it seems that 3.0 is quite a bit different from <3.0 and we might have some compatibility issues if we support pre-3.0 versions. Do you have an opinion / better sense of this?

Reply:
The ecosystem is evolving fast. In fact I can't even find <3.0 docs anymore (although I did a quick search only). Maybe it would be good to force `>=3.0.0` (not `> 3.0.0` as `3.0.0` is newest? or maybe I'm wrong and `> 3.0.0` includes 3.0.0): people will be using this integration primarily for _new models_, I presume, not to convert their existing scripts/pipelines.

Reply:
Yep. `>=3.0.0` would be good I think.

Reply:
If people ask for a wider range we can always consider it as and when people use the integration. And of course there's always the option of having a custom materializer etc.

Reply:
Done :)